File: java/src/main/java/com/gushenge/atools/util/DateUtils.java
package com.gushenge.atools.util;


import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;

public class DateUtils {
    public String stampToDate(int s){
        String res;
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
        Long lt  = Long.parseLong(Integer.toString(s)) * 1000;
        Date date = new Date(lt);
        res = simpleDateFormat.format(date);
        return res;
    }
    public String dateToStamp(String s) throws ParseException{
        String res;
        SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
        Date date = simpleDateFormat.parse(s);
        long ts = date != null ? date.getTime() : 0;
        res = String.valueOf(ts/1000);
        return res;
    }
}

File: kotlin/src/main/java/com/gushenge/atools/util/DateUtils.kt
package com.gushenge.atools.util

import java.text.ParseException
import java.text.SimpleDateFormat
import java.util.Date

object DateUtils {
    /*
     * 将时间戳转换为时间
     */
    val simpleDateFormat = SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
    fun stampToDate(s: Int): String {
        val lt  = s.toLong() * 1000
        val date = Date(lt)
        return simpleDateFormat.format(date)
    }

    /*
     * 将时间转换为时间戳
     */
    @Throws(ParseException::class)
    fun dateToStamp(s: String): String {
        val date = simpleDateFormat.parse(s)
        val ts = date!!.time
        val stamp = ts/1000
        return stamp.toString()
    }
    fun getDate() : String{
        //获取当前时间
        val date = Date(System.currentTimeMillis())
        return simpleDateFormat.format(date)
    }

    fun getStamp():String{
        val timeStamp = System.currentTimeMillis()
        return timeStamp.toString().substring(0, 10)
    }
}
